frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.ts

```diff
@@ -147,6 +181,57 @@ export class ChallengesettingsComponent implements OnInit, OnDestroy {
     }
   }
 
+    /**
+   * Called when a phase is selected (from child component)
+   */
+    phaseSelected() {
```

Comment:
@gautamjajoo can you fix the indentation here

Reply:
Done!